e2e/test/scenarios/embedding/embedding-full-app.cy.spec.js

```diff
@@ -135,15 +137,14 @@ describeEE("scenarios > embedding > full app", () => {
   });
 
   describe("browse data", () => {
-    it("should hide the top nav when nothing is shown", () => {
+    it("should hide the side nav when nothing is shown", () => {
       visitFullAppEmbeddingUrl({
         url: "/browse",
         qs: { side_nav: false, logo: false },
       });
-      // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-      cy.findByText("Our data").should("be.visible");
-      // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-      cy.findByText("Our analytics").should("not.exist");
+      cy.findByRole("heading", { name: /Our data/ }).should("be.visible");
+      cy.findByRole("treeitem", { name: /Our data/ }).should("not.exist");
+      cy.findByRole("treeitem", { name: "Our analytics" }).should("not.exist");
       appBar().should("not.exist");
```

Comment:
I'm not totally sure I've understood what this test is about

Reply:
IIRC, @WiNloSt originally wrote these tests.
I think the test is doing what it says on the tin - if we hide the side navigation **and** hide the logo, we want to make sure the top bar is not rendered.

There is a test below that explicitly makes sure nav bar is hidden via param.

Reply:
Thank you for helping me understand. I've reverted that change

Reply:
@nemanjaglumac got it exactly right. That test was added during the implementation of hiding Metabase logo.

For context if you're interested:
- [PR](https://github.com/metabase/metabase/pull/27391)
- [Issue](https://github.com/metabase/metabase/issues/27293)